05-onchain.md

```diff
@@ -178,25 +178,25 @@ the other node's `to_self_delay` field). Where relevant, this is noted below.
 When a node sees *its own* commitment transaction:
 
 1. `to_local` output: A node SHOULD spend this output to a convenient address.
-   A node MUST wait until the `OP_CHECKSEQUENCEVERIFY` delay has passed (as specified by the other
+   A node MUST wait until the `OP_CHECKSEQUENCEVERIFY` delay has passed (as specified by the remote
    node's `to_self_delay` field) before spending the output. If the
    output is spent (as recommended), the output is *resolved* by the spending
    transaction, otherwise it is considered *resolved* by the commitment transaction itself.
 2. `to_remote` output: No action required, this output is considered *resolved*
    by the commitment transaction itself.
-3. HTLCs offered by this node: See "HTLC Output Handling: Our Commitment, Our Offers" below.
-4. HTLCs offered by the other node: See "HTLC Output Handling: Our Commitment, Their Offers" below.
+3. HTLCs offered by this node: See "HTLC Output Handling: Local Commitment, Local Offers" below.
+4. HTLCs offered by the remote node: See "HTLC Output Handling: Local Commitment, Remote Offers" below.
```

Comment:
We should probably use a #-link for these two...

Reply:
ok